bioptim/examples/getting_started/custom_dynamics.py

```diff
@@ -68,7 +68,8 @@ def custom_dynamic(
     dq = DynamicsFunctions.compute_qdot(nlp, q, qdot) * my_additional_factor
     ddq = nlp.model.ForwardDynamics(q, qdot, tau).to_mx()
 
-    return dq, ddq
+    defects = MX.zeros(1)
+    return (dq, ddq), defects
```

Comment:
As stated before, this should not be mandatory. 
So either we add a flag to inform bioptim that this return is in the implicit or explicit form (probably in `dynamics.add`) or we interpret the output, which seems a bit harder (since it is already a tuple)

Reply:
also, the function may need xdot as an input.